api/models.py

```diff
@@ -15,20 +15,30 @@ def __str__(self):
 class Event(models.Model):
     """ A disaster, which could cover multiple countries """
 
+    eid = models.IntegerField(null=True)
     name = models.CharField(max_length=100)
+    dtype = models.ForeignKey(DisasterType, null=True)
     summary = models.TextField(blank=True)
+    status = models.CharField(max_length=30, blank=True)
+    region = models.CharField(max_length=100, blank=True)
+    code = models.CharField(max_length=20, null=True)
+
     created_at = models.DateTimeField(auto_now_add=True)
-    # type = models.ForeignKey(DisasterType)
 
     def countries(self):
         """ Get countries from all appeals and field reports in this disaster """
-        #countries1 = [appeal.country for appeal in self.appeal_set.all()]
-        countries = [country for fr in self.fieldreport_set.all() for country in fr.countries.all()]
-        #flat_list = [item for sublist in l for item in sublist]
-
-        #countries2 = [fr.country for fr in self.fieldreport_set.all()]
+        countries = [country for fr in self.fieldreport_set.all() for country in fr.countries.all()] + \
```

Comment:
I have a feeling this *might* be superfluous since the field report countries should match the appeal countries, but I like the insurance of doing it this way